api/controller/block.ts

```diff
@@ -0,0 +1,100 @@
+import {BlockService} from 'api/service/block';
+import Response from 'shared/model/response';
+import {Block} from 'shared/model/block';
+import {BlockRepo} from 'api/repository/block';
+
+// exists in core/service
+const BlockReward = require('../../logic/blockReward.js');
+const constants = require('../../helpers/constants.js');
+const schema = require('../../schema/blocks.js');
+
+@Controller('/blocks')
+export class BlockController {
+    private blockReward = new BlockReward();
+    private blockService = new BlockService();
+    private blockRepo = new BlockRepo();
+
+    constructor() {}
```

Comment:
Initialization of components place here, in constructor.

Reply:
Not fixed

Reply:
Will be fixed while moving all related services to singletons.